draft-ietf-quic-transport.md

```diff
@@ -2403,6 +2403,15 @@ that it sends.
 Strategies and implications of the frequency of generating acknowledgments are
 discussed in more detail in {{QUIC-RECOVERY}}.
 
+# Special Considerations for Packetization Layer PMTU Discovery
+
+The PADDING frame provides a useful option for PMTU probe packets that does not
+exist in other transports. QUIC Packets with no frames other than PADDING must be
+acknowledged, but need not be retransmitted if lost. PADDING probe packets, if
```

Comment:
We're trying to avoid talking about retransmission of packets, instead concentrating on the information content.  In this case, you could say "QUIC packets that only include PADDING frames cause the receiver to generate an acknowledgment, but PADDING frames don't contain information that needs to be reliably delivered."  Or something like that.